### PR TITLE
test: [M3-8149] - Add test for Linode create error flows

### PR DESCRIPTION
**Pull request by AzureLatte**
## Description 📝
Add Cypress integration tests using mock API data to confirm that Cloud Manager behaves gracefully when errors occur during the Linode create flow.

## Changes  🔄
- Add util function to mock error response.
- Add new test to confirm error handling behaviour during the Linode create flow.

## How to test 🧪
```
yarn cy:run -s "cypress/e2e/core/linodes/create-linode.spec.ts"
```
## As an Author I have considered 🤔

*Check all that apply*

- [x] 👀 Doing a self review
- [ ] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [ ] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a changeset
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/manager/cypress/e2e/core/linodes/create-linode.spec.ts [hunk 1]
```diff
@@ -341,4 +347,56 @@ describe('Create Linode', () => {
     fbtVisible(linodeLabel);
     cy.contains('RUNNING', { timeout: 300000 }).should('be.visible');
   });
+
+  /*
+   * - Confirms error message can show up during Linode create flow.
+   * - Confirms Linode can be created after retry.
+   */
+  it('unexpected error show up during Linode create flow', () => {
```
Comment (dwiley-akamai) [inline packages/manager/cypress/e2e/core/linodes/create-linode.spec.ts:355]:
```suggestion
  it('shows unexpected error during Linode create flow', () => {
```

**Review by dwiley-akamai — APPROVED** [review]
Test passes in CI ✅ (I observed timeout failures locally)

**Review by bnussman-akamai — APPROVED** [review]

File: packages/manager/cypress/e2e/core/linodes/create-linode.spec.ts (hunk 1 above)
Comment (AzureLatte) [reply packages/manager/cypress/e2e/core/linodes/create-linode.spec.ts]: Updated!

**Comment by AzureLatte** [comment]
E2E failure is not related with this change

**Pull request merged**